hw/fpga/fpga_configuration.tcl

```diff
@@ -28,10 +27,10 @@ if {[info exists VERSION] == 0} {
 }
 
 # Path to rtl
-if {$HW_LATEST} {
-  set rtlDir $fpgaDir/../latest/rtl
+if {$RTL_VERSION} {
+  set rtlDir $RTL_DIR/../$RTL_VERSION/rtl
```

Comment:
This intermediate step is probably broken because RTL_DIR isn't set.

Reply:
Updated